### PR TITLE
Task 10 patterns

### PR DESCRIPTION
**Pull request by suffergram**

### CODE REVIEW

File: src/assets/fonts/Montserrat.ttf
Comment (souljja) [inline src/assets/fonts/Montserrat.ttf:1]: Let's convert this font to woff2.

File: webpack.config.js [hunk 1]
```diff
@@ -43,6 +43,15 @@ module.exports = (argv) => {
           test: /\.(woff|woff2|eot|ttf|otf)$/i,
           type: 'asset/resource',
         },
+        {
+          test: /\.(png|svg|jpg|jpeg|gif|webp)$/i,
+          use: ['file-loader', 'webp-loader'],
```
Comment (souljja) [inline webpack.config.js]: Is this web-p loader needed? Can't it be done via `type: 'asset/resource'` ?

**Review by souljja — APPROVED** [review]

File: src/assets/fonts/Montserrat.ttf
Comment (suffergram) [reply src/assets/fonts/Montserrat.ttf:1]: Converted to woff2.

File: webpack.config.js (hunk 1 above)
Comment (suffergram) [reply webpack.config.js]: Changed it to `type: 'asset/resource'`.

**Pull request merged**